tests/infra/node.py

```diff
@@ -244,43 +244,19 @@ def wait_for_node_to_join(self, timeout=3):
     def get_ledger(self):
         return self.remote.get_ledger()
 
-    def user_client(self, user_id=0, **kwargs):
+    def client(self, identity=None, **kwargs):
+        akwargs = {
+            "cert": os.path.join(self.common_dir, f"{identity}_cert.pem") if identity else None,
+            "key": os.path.join(self.common_dir, f"{identity}_privk.pem") if identity else None,
+            "ca": os.path.join(self.common_dir, "networkcert.pem"),
+            "description": f"node {self.node_id} as {identity}",
```

Comment:
Minor, just a suggestion, open to a better term here (I think we've used "anonymous" elsewhere?)

```suggestion
            "description": f"node {self.node_id} as {identity or "anyone"}",
```

Reply:
That'll show "None" right now. I think I'd mildly prefer "unauthenticated" rather than "anyone" (there _is_ a caller, so it feels more like "someone" but we don't know who?).